syntax = "proto3";

package greet;

service Greeting
{
  rpc HelloEcho (HelloRequest) returns (HelloResponse) {}
  rpc RegistrateUser (Registrate) returns (HelloResponse) {}
}

message HelloRequest
{
  string login = 1;
  string password = 2;
}

message Registrate
{
  string login = 1;
  string password = 2;
}

message HelloResponse
{
  string message = 1;
}